### PR TITLE
Another bunch of editorial fixes

### PR DESCRIPTION
**Pull request by dontcallmedom**
in the same spirit and process as #333

### CODE REVIEW

File: webrtc.html [hunk 1]
```diff
@@ -3043,6 +3035,39 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
           </dd>
 
         </dl>
+        <dl class="idl" title="dictionary RTCRtpTransceiverInit">
```
Comment (alvestrand) [inline webrtc.html]: I don't get why this has moved. Shouldn't it stay with RTCRtpTransceiver?

Comment (dontcallmedom) [inline webrtc.html]: I moved it closer to where it is being used (in `addTransceiver`); at the moment, it is with `RTCRtpTransceiver`, but nothing in that interface makes use of `RTCRtpTransceiverInit`, with the result that the reader is left puzzled as to what it is used for.

**Comment by alvestrand** [comment]
@adam-be says he can merge once you've rebased it if you want a second pair of eyes.
Otherwise, just merge.

**Comment by dontcallmedom** [comment]
rebased; assigning to @adam-be  for a last check.

**Comment by adam-be** [comment]
Ops. I missed this. Now we might as well merge this one at the editor's meeting later today.

**Comment by alvestrand** [comment]
Oops - we let other PRs disturb it again. Please rebase and merge immediately!

**Pull request merged**